GeometryTools/Rectangle.swift

```diff
@@ -234,7 +234,6 @@ extension Rectangle: CustomStringConvertible {
     // MARK: CustomStringConvertible
 
     public var description: String {
-        return "Rectangle[ origin: (\(origin.x), \(origin.y)), "
-            + "width: \(size.width), height: \(size.height) ]"
+        return "Rectangle[ origin: \(origin), width: \(size.width), height: \(size.height) ]"
```

Comment:
Actually, I don't think need the `Rectangle` description. Particularly if we are using `dump`, it will tell us what it is.

I would say, make the result: `origin: \(origin), width: \(size.width), height: \(size:height)`.